### DISCUSSION TITLE
Unable to download issue

### DISCUSSION THREAD

**Discussion by MkMinhaj** [aniyomiorg/aniyomi#1231] [General]
Hey, I can not download any episode. When I hit the download button it turns into an exclamation mark (!).
![403417622_915493473271790_1661397879715890306_n](https://github.com/aniyomiorg/aniyomi/assets/152235455/0bd25bed-7ead-47ad-8ca3-6244d20ece3f)
Like this

**Comment by LittleFreak** [comment]
Have you tried other hoster within gogoanime, or even another source yet?
